src/Debugger/Impl/DataInspection/RValueProperties.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.R.DataInspection;
+using Microsoft.R.StackTracing;
+using static System.FormattableString;
+
+namespace Microsoft.R.DataInspection {
+    /// <summary>
+    /// Used to specify properties of <see cref="IRValueInfo"/> to fill when evaluating expressions using
+    /// <see cref="RSessionExtensions.TryEvaluateAndDescribeAsync"/> and its various wrappers.
+    /// </summary>
+    [Flags]
+    public enum RValueProperties : ulong {
```

Comment:
Well, it is not ideal but name at least looks better than other options. Perhaps RValueAttributes or RValueInfoAttributes? sort of like FileInfo I guess.